packages/pn-pa-webapp/src/App.tsx

```diff
@@ -185,6 +186,9 @@ const ActualApp = () => {
     if (sessionToken) {
       void dispatch(getCurrentAppStatus());
       void dispatch(getInstitutions());
+      // Temporary fix for multi-language support
+      // Remove lang from sessionStorage in order to makes Footer works properly
+      sessionStorage.removeItem('lang');
```

Comment:
Otherwise, upon refreshing the page, the language will revert to Italian, but the footer, reading the value from sessionStorage, will display the previously selected language.

Reply:
And we can't manage the refresh as for the others webapp?